### ISSUE TITLE
TypeError: Cannot read property 'line' of undefined

### ISSUE THREAD

**Issue by curtisblackwell** [AtomLinter/linter-js-yaml#9]
This is different from #8, but it may be related. I get this error on save, but not when I switch tabs or anything else.

```
TypeError: Cannot read property 'line' of undefined
    at Object.provider.processMessage (/Users/[REDACTED]/.atom/packages/linter-js-yaml/lib/linter-js-yaml.coffee:17:32)
    at /Users/[REDACTED]/.atom/packages/linter-js-yaml/lib/linter-js-yaml.coffee:37:38
    at Object.provider.lint (/Users/[REDACTED]/.atom/packages/linter-js-yaml/lib/linter-js-yaml.coffee:26:18)
    at /Users/[REDACTED]/.atom/packages/linter/lib/editor-linter.coffee:110:37
    at currentLinter (/Users/[REDACTED]/.atom/packages/linter/lib/editor-linter.coffee:109:20)
    at /Users/[REDACTED]/.atom/packages/linter/lib/editor-linter.coffee:132:34
    at Set.forEach (native)
    at EditorLinter.triggerLinters (/Users/[REDACTED]/.atom/packages/linter/lib/editor-linter.coffee:99:32)
    at /Users/[REDACTED]/.atom/packages/linter/lib/editor-linter.coffee:87:36
```

I see that it happens when the YAML file ends with `---`. I have to have the `---` because it's how the CMS I'm working with handles user files.

**Comment by jmccarty3** [comment]
I see a similar issue anytime i put in "---" lines into the file.

**Comment by praveen12bnitt** [comment]
Even i see the same error.

**Comment by steelbrain** [comment]
Fixed by #13
